### PR TITLE
[MODCON-53] Cover missed karate scenarios for Consortia functionality

### PR DESCRIPTION
**Pull request by bekhzodkurbonboev**
## Purpose
Continuation of [MODCON-53](https://issues.folio.org/browse/MODCON-53) Cover missed karate scenarios for Consortia functionality:
Added test case: _If we DELETE real user all records related to this user should be deleted for both 'central' (isCentral=true) and institutional (isCentral=false) tenants_

## Approach
_How does this change fulfill the purpose?_

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

**Comment by azizbekxm** [comment]
image

File: acquisitions/src/main/resources/thunderjet/mod-consortia/features/user-tenant-associations.feature [hunk 1]
```diff
@@ -486,4 +487,168 @@ Feature: Consortia User Tenant associations api tests
...
+    # DELETE 'centralUser1'
+    * call read(login) consortiaAdmin
+    * configure headers = { 'Content-Type': 'application/json', 'Accept': 'text/plain' }
+    Given path 'users', centralUser1.id
+    And headers {'x-okapi-tenant':'#(tenant)', 'x-okapi-token':'#(okapitoken)' }
+    When method DELETE
+    Then status 204
+
+    # 1. verify there is no record in 'users' table in 'central_mod_users' for 'centralUser1'
+    * configure headers = { 'Content-Type': 'application/json', 'Accept': 'application/json' }
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/mod-consortia/features/user-tenant-associations.feature:503]: Is test case missed here?

Comment (bekhzodkurbonboev) [reply acquisitions/src/main/resources/thunderjet/mod-consortia/features/user-tenant-associations.feature:503]: This is a functionality of 'mod-users'. Here purpose of this is to indicate we have this case also.

Comment (SerhiiNosko) [reply acquisitions/src/main/resources/thunderjet/mod-consortia/features/user-tenant-associations.feature:503]: okay got it

**Review by SerhiiNosko — APPROVED** [review]

**Review by azizbekxm — APPROVED** [review]

**Pull request merged**